pkg/operator/resources/cluster/apiserver.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2018 The CDI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	rbacv1beta1 "k8s.io/api/rbac/v1beta1"
```

Comment:
I was just bringing over the exact resources from the manifest.  I'm not sure if there are any differences between the versions for the resources we use in this package.  But if you think it would be wise, I can bump the versions.